charts/invenio/templates/_helpers.tpl

```diff
@@ -50,6 +50,27 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
+{{/*
+Return the proper Invenio image name
+*/}}
+{{- define "invenio.image" -}}
+{{- $registryName := .Values.image.registry -}}
+{{- $repositoryName := .Values.image.repository -}}
```

Comment:
I think both of these should be marked as required. Depends on how we want to handle Docker Hub, I guess. See my other comment for that discussion.